shell/components/SortableTable/selection.js

```diff
@@ -531,7 +531,7 @@ export default {
     },
 
     clearSelection() {
-      this.update([], this.selectedRows);
+      this.update([], [...this.selectedRows]);
```

Comment:
`update` second param toRemove is entries to remove from the selection. it runs through those and removes it's entries from this.selectedRows. Without spread it in affect removes entries from itself, so later on there's nothing in toRemove to remove (though it has been removed from this.selectedRows). by spreading it means items are still removed from this.selectedRows but still exist in toRemove